styx-scheduler-service/src/main/java/com/spotify/styx/state/StateUtil.java

```diff
@@ -110,21 +111,24 @@ public static Set<String> workflowResources(boolean globalConcurrenyEnabled,
   }
 
   public static boolean hasTimedOut(Optional<Workflow> workflowOpt, RunState runState, Instant instant,
-                                     Duration timeout) {
+                                     Duration defaultTimeout, Duration maxRunningTimeout) {
```

Comment:
Maybe `defaultTimeout` can be renamed to `runStateTimeout`? It's not really a default.

Reply:
fixed